docs/source.rst

```diff
@@ -18,77 +18,54 @@ Source Guide
    methods, including email to Freedom of the Press Foundation.
 
 What is SecureDrop?
----------------------------
+-------------------
 
-Dozens of news organizations — from *ProPublica* to *The New York Times* — use
-SecureDrop to accept tips securely and anonymously. You can reach out and share
-files, and messages, but for real anonymity, it’s important to take some extra
-precautions. This resource will describe things you can do to help protect your
-anonymity when using SecureDrop.
+SecureDrop is a tool that news organizations and NGOs use that enables secure and anonymous communication between whistleblowers and journalists. No personal information is collected; SecureDrop's servers are encrypted, and SecureDrop is not a “cloud” service. If you don’t have sensitive information to send to a news organization, it may be okay to use a traditional computer when reaching out. 
```

Comment:
This "traditional computer" language was previously used in the context where buying an extra computer was suggested; without that context I would suggest just saying "traditional methods such as phone or email".